### PR TITLE
[QI2-853] Add number of shots

### PR DESCRIPTION
**Pull request by koffie**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Created by https://www.toptal.com/developers/gitignore/api/python,intellij+all,visualstudiocode,vagrant
 # Edit at https://www.toptal.com/developers/gitignore?templates=python,intellij+all,visualstudiocode,vagrant
 
+### custom ignores
+scratch
```
Comment (oschusler) [inline .gitignore]: Could you please add this to the bottom of the file?

File: examples/simple.py [hunk 2]
```diff
@@ -5,18 +5,22 @@
...
+    with Circuit(platform_name="spin-2", program_name="prgm1") as c:
+        k = c.init_kernel("new_kernel", 2)
+        k.x(0)
+        k.hadamard(1)
+        k.measure(0)
+        k.measure(1)
 
-backend = RemoteBackend()
+    print(c.content)
+    print(f"number of qubits: {c.max_number_of_qubits}")
```
Comment (oschusler) [inline examples/simple.py]: Why do you add this statement?

Comment (koffie) [reply examples/simple.py]: Ah should have removed these again. This was for debugging some other weird behaviour:

```
# Generated by OpenQL 0.12.2 for program prgm1
version 1.0

qubits 10


.new_kernel
    x q[0]
    h q[1]
    measure q[0]
    measure q[1]

number of qubits: 2
Execution time in seconds: 0.3009359836578369
job_id: 1035
```
So even though we explicitly say that there are 2 qubits. OpenQL still compiles it to 10 qubits. I'll remove these debug statements again.

File: .gitignore (hunk 1 above)
Comment (koffie) [reply .gitignore]: Done

File: examples/simple.py (hunk 2 above)
Comment (oschusler) [reply examples/simple.py]: That's "correct". OpenQL takes the maximum amount of qubits for that specific "backend".

**Review by oschusler — APPROVED** [review]

**Pull request merged**